.github/workflows/docker-build.yml

```
@@ -46,6 +46,14 @@ jobs:
                 id: buildx
                 uses: docker/setup-buildx-action@v1
 
+            -   name: Cache Docker layers
```

```suggestion
            -   name: Cache Docker layers
                if: github.event_name != 'pull_request'
```
Don't allow PRs to break the cache.

---

(Additional changes below required as well.)

---

But don't we want the cache to be updated by PR so that when the PR is merged to main the image layers are already in cache?

---

Hm yeah, I suppose.  But then anyone could open a PR and break the cache?  Do we have actions configured to run for any user, or is approval required for first-time contributors?

---

Approval is required